lib/blueprinter/view.rb

```diff
@@ -4,12 +4,12 @@ module Blueprinter
   class View
     attr_reader :excluded_field_names, :fields, :included_view_names, :name, :transformers, :definition_order
 
-    def initialize(name, fields: {}, included_view_names: [], excluded_view_names: [],transformers: [])
+    def initialize(name, fields: {}, included_view_names: [], excluded_view_names: [], transformers: [Blueprinter.configuration.transform_default].compact)
```

Comment:
The way this would work, the config-set transformer is really more of "base transform" than a default transform.
i.e. in this approach, the transform is not overwritten by the view-level transform.
I think it may be more flexible if the `transform_default` was made something like `default_transformers = []` and was over-writeable. So instead, perhaps implement the `transformers` accessor as:
```
def transformers
  transformers.empty? Blueprinter.configuration.default_transformers : transformers
end
```

That **pro** here is that a user can specify any amount of base transforms to be ran by default, but if a user wants to override this at the view level, they can.

The **con** here is if the user wants to **add** to the default_transforms instead of override, they'd need to specify all the default transforms again in addition to the added one(s).

I think this con may still be ok as the alternative would mean there isn't really a great way for users to override the default_transformers.

Thoughts @philipqnguyen and @supremebeing7 ?

Reply:
Ah you're right, I had actually tested that scenario locally but I'm realizing I didn't test it explicitly enough. So yes, I would agree on that being the preferred behavior. I will go ahead and implement that, unless/until @philipqnguyen has a different opinion.